### PR TITLE
fix path to hooks

### PR DESCRIPTION
**Pull request by sallyom**
Signed-off-by: Sally O'Malley <somalley@redhat.com>

### CODE REVIEW

**Comment by sallyom** [comment]
:) this works

File: dockerhooks/dockerhooks.go [hunk 1]
```diff
@@ -13,9 +13,6 @@ import (
 
 func getHooks() (string, []os.FileInfo, error) {
 	hookPath := os.Getenv("DOCKER_HOOKS_PATH")
-	if hookPath != "" {
```
Comment (mrunalp) [inline dockerhooks/dockerhooks.go]: I think could just invert the check here instead.

Comment (sallyom) [inline dockerhooks/dockerhooks.go]: this works, too ;)

Comment (rhatdan) [inline dockerhooks/dockerhooks.go]: Yes I fixed this and updated the patch.  Sorry about not being able to fix.  I also found a couple of other problems which I fixed.  

Please take one last look and we will submit upstream on Monday.

**Pull request closed without merging**